### ISSUE TITLE
Unable to play Songs Folder

### ISSUE THREAD

**Issue by brnwlshubh** [YajanaRao/Serenity#123]
Meditation & Podcasts are working fine but in Songs tab/folder serenity app crashes instead of song playing.

**Comment by brnwlshubh** [comment]
Do check from your side.

**Comment by brnwlshubh** [comment]
More precise reason for app crash is "Play All" button

**Comment by YajanaRao** [comment]
This issue is fixed @brnwlshubh . I will create a release soon. The fix is in develop branch. You can find the apk here: https://github.com/YajanaRao/Serenity/actions

**Comment by brnwlshubh** [comment]
Did you fixed #PlayAll from all tabs(Songs Meditation & Podcasts)?

**Comment by YajanaRao** [comment]
Yes. This is not released yet. You can find apk here: https://github.com/YajanaRao/Serenity/actions

**Comment by brnwlshubh** [comment]
@YajanaRao Can you add this app on Fdroid or Izzy?

**Comment by YajanaRao** [comment]
I will look into that @brnwlshubh . I have not use them before.
